src/sentry/static/sentry/app/views/releaseDetails.jsx

```diff
@@ -18,6 +21,54 @@ import TimeSince from 'app/components/timeSince';
 import Version from 'app/components/version';
 import withEnvironmentInQueryString from 'app/utils/withEnvironmentInQueryString';
 
+class ReleaseDetailsActions extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    release: PropTypes.object.isRequired,
```

Comment:
Just an fyi, we have prop type shapes in `app/sentryTypes`, but not one for a Release so no worries here.

Reply:
That'd be great

Reply:
I can do it as a follow up to this pull request.